packages/core/src/application.ts

```diff
@@ -192,6 +192,16 @@ export class Application extends Context {
     const instance = this.getSync<Component>(componentKey);
     mountComponent(this, instance);
   }
+
+  /**
+   * Set application metadata. By default, the content of `package.json` is
```

Comment:
"By defaults" implies to me that `package.json` metadata is populated by Application constructor, which is not true.

Can we make it more clear please that this metadata is empty by default but `@loopback/boot` populates is from `package.json`?